zipkin-lens/package.json

```diff
@@ -26,13 +26,14 @@
   "author": "https://gitter.im/openzipkin/zipkin",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@babel/cli": "^7.5.0",
+    "@babel/core": "^7.5.4",
+    "@babel/preset-env": "^7.5.4",
+    "@babel/preset-react": "^7.0.0",
     "@fortawesome/fontawesome-free": "^5.4.1",
-    "babel-core": "^6.26.3",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
This package is necessary 

↓ Jest 23.6's document
https://github.com/facebook/jest/blob/770799779fc853ab6285727e8938efd06e694d5f/README.md#using-babel